docs/nodes/full-node/network-identity-fullnode.md

```diff
@@ -1,42 +1,45 @@
 ---
-title: "Network Identity For Fullnode"
+title: "Network Identity For PFNs"
 slug: "network-identity-fullnode"
 ---
 
-# Network Identity For Fullnode
+# Network Identity For PFNs
 
-Fullnodes will automatically start up with a randomly generated network identity. This works well for regular fullnodes. However:
+:::danger
+Generating an identity should only be done for PFNs, and not for validator fullnodes (VFNs). VFNs should always the
+identities generated when the validator and VFN pair were originally deployed.
+:::
```

Comment:
Rephrase as: VNs and VFNs have their identities initialized when deployed onto the blockchain, after which the identities must be considered immutable because they're embedded into the on-chain config.
PFN identities are more ephemeral and can be regenerated, keeping in mind that any change in identity will have to be propagated to any peer allowlist(s) that included the previous ID.

Reply:
Sounds good -- used a mixture of the two 😄 (I don't want to overly share information in the snippet).